kubernetes/samples/scripts/create-weblogic-domain/model-in-image/utils/patch-enable-online-update.sh

```diff
@@ -54,7 +54,7 @@ set -o pipefail
 
 echo "@@ Info: Patching domain '${DOMAIN_UID}' in namespace '${DOMAIN_NAMESPACE}' to enable onlineUpdate to 'true'."
 kubectl -n ${DOMAIN_NAMESPACE} patch domain ${DOMAIN_UID} --type='json' \
-  -p='[{"op": "replace", "path": "/spec/configuration/model/onlineUpdate", "value": {"enabled" : 'true'} }]'
+  -p='[{"op": "add", "path": "/spec/configuration/model/onlineUpdate", "value": {"enabled" : 'true'} }]'
```

Comment:
(1) Why is this change needed?  "replace" worked for me in all three cases:
```
   model:

   model:
      onlineUpdate:

   model:
      onlineUpdate: false
``` 

(2) The documentation at ./content/samples/simple/domains/model-in-image/update4.md also needs to be updated to match whatever we change here.  (The doc mentions the same 'patch domain' command.)

Reply:
ok, then we don't need this anymore, I was testing few days ago and changing add from replace let the integration tests passed.  I will back this out

Reply:
revered